### PR TITLE
Feat : Added WishlistPage

### LINKED ISSUE #17: update : wishlist.html
fixed typos in wishlist page

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. wishlist page
WishlistPage

### CODE REVIEW

File: css/screens/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* css for wishlist */
+
+/* css for wishlist section */
```
Comment (rahulcs754) [inline css/screens/wishlist.css]: don't write unnecessary comment

**Review by rahulcs754 — APPROVED** [review]
I looked your code and that is good but some small suggestion I will give please check

File: css/screens/wishlist.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/screens/wishlist.css]: sure ...removed

File: css/screens/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* css for wishlist */
+
```
Comment (rohit8686) [inline css/screens/wishlist.css:2]: can remove this empty lines

File: css/screens/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+.wishlist-contents{
+    flex-wrap: wrap;
+    padding: 2rem 0rem;
```
Comment (rohit8686) [inline css/screens/wishlist.css]: it is better not to give units to 0 value, other than these everything looks good !

**Review by rohit8686 — APPROVED** [review]

**Pull request merged**